src/js/core/support.js

```diff
@@ -222,6 +223,17 @@
 				return div.getBoundingClientRect !== undefined;
 			}
 
+			ieVersion = (function getIEVersion() {
```

Comment:
Could this be a member of browser object?
```
browser {
  isIE: func()
  ieVersion: func()
}
```

Reply:
I don't want to change API.
Now, `ieVersion` is called once.
Version of IE is compared only in one place. `this.browser.ie >= 7`

Reply:
How about just adding ieVersion to browser object instead of defining it separately? It looks wired to have ieVersion() function where we have browser object with ie property.

Reply:
As I wrote before I don't want to change API. I just try to make refactoring for fix SVACE issue.